### DISCUSSION TITLE
S23 Ultra Settings for stability

### DISCUSSION THREAD

**Discussion by markjcw** [BimmerGestalt/AAIdrive#709] [Show and tell]
Hi.. I have attached screenshots - for S23 Ultra navigate to Battery device care > memory > excluded apps then add the aadrive app and mini/bmw app to be excluded from memory usage check. This should stop Android killing app in background.

Since I have implemented this setting aadrive does not disconnect and also automatically connects when first starting car too.

Hope this works for anyone having connection issue on S23 series.

It's a great app.

![Screenshot_20230607_084318_Device care.png](https://github.com/BimmerGestalt/AAIdrive/assets/128420640/8f893ecf-b8d8-40c0-886e-3ac1ad56479e)

![Screenshot_20230607_084534_Device care.jpg](https://github.com/BimmerGestalt/AAIdrive/assets/128420640/f7872281-b726-4708-88fb-4962b8c23d87)

**Comment by jimgandhi** [comment]
thanks!  I just read this and set on my S23Ultra.  I was wondering if there was a setting like this.
